src/sparse/impl/KokkosSparse_spmv_impl.hpp

```diff
@@ -985,10 +986,10 @@ struct SPMV_MV_OpenMP_Functor {
 			  const XVector m_x_,
 			  const value_type beta_,
 			  const YVector m_y_,
-			  const ordinal_type rows_per_team_) :
+			  const int rows_per_thread_) :
```

Comment:
The right type should be `ordinal_type`, if that's what you're using to store the number of rows.  What if there's only one thread?  Then this number needs to store the number of rows.